### ISSUE TITLE
new VWF multimer size term

### ISSUE THREAD

**Issue by caripark** [obophenotype/mammalian-phenotype-ontology#4259]
Preferred term label (e.g., asplenia)
Increased plasma (or circulating) von Willebrand factor multimer size

Synonyms (e.g., absent spleen)
Increased plasma (or circulating) VWF multimer size

Suggested parent term (Please look in the hierarchy of the MP using the MP Browser or other browsers such as OLS and find the place in the MP where the new term best fits)
Abnormal blood coagulation (HP has “abnormality of coagulation” > “abnormality of the coagulation cascade” > “abnormality of von Willebrand factor” > “abnormal von Willebrand factor multimer distribution” > “abnormal presence of ultra-large von Willebrand factor multimers” (HP:0030133))

Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).
Enlarged multimers in the blood comprising the large multimeric glycoprotein produced by vascular endothelial cells and megakaryocytes that chaperones blood coagulation cofactor factor VIII and tethers platelets to the injured blood vessel wall
MP:0031163, PMID:41915891

If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)
0000-0002-2346-5201

**Comment by sbello** [comment]
Assuming we treat abnormal blood homeostasis as covering all abnormal blood chemistry which is essentially what we've been doing by sticking all the level terms here, then we can put this new term there.

From PMID:16368888
VWF is a large glycoprotein that mediates platelet adhesion to vascular lesions. It is mainly synthesized in endothelial cells and secreted into the blood as “unusually large” VWF (UL-VWF) multimers, the highly active forms for platelet adhesion and aggregation.11,12 ADAMTS13 cleaves UL-VWF multimers into smaller forms under flow, limiting platelet thrombus formation under normal conditions.

Looking at blood coagulation in GO, def
The sequential process in which the multiple coagulation factors of the blood interact, ultimately resulting in the formation of an insoluble fibrin clot; it may be divided into three stages: stage 1, the formation of intrinsic and extrinsic prothrombin converting principle; stage 2, the formation of thrombin; stage 3, the formation of stable fibrin polymers.
has a part of child terms platelet aggregation and platelet activation 

Current MP definition of abnormal blood coagulation is 'altered ability or inability of the blood to clot'
This also sits under blood homeostasis
If we stretch the definition of blood coagulation to include abnormal von Willebrand factor multimer size then should we also include the level terms here?
What about the other coagulation factors?

Note PMID:16368888 refers to VWF level measurement as "Plasma VWF antigen was measured by ELISA using antibodies against
human VWF"  need to look at existing annotations as for this paper this was annotated to [abnormal blood coagulation](https://www.informatics.jax.org/vocab/mp_ontology/MP:0002551) and not VWF level

- consider adding plasma vWF antigen level as a synonym to existing terms

PMID:23863626 reports VWF levels in platelets and megakaryocytes (located in bone marrow) - this is currently annotated to morph terms for these cells.

PMID:[8490171](https://pubmed.ncbi.nlm.nih.gov/8490171/) reports platelet levels annotated to platelet alpha granule morphology 

 MGI:3697681 (genotype) has an annotation for VWF level in placental vasculature

MGI:5829681 (genotype) reports decreased VWF secretion by platelets but 'normal bleeding times' 

MGI:3687213 (genotype) 'reduced numbers of von Willebrand factor-positive blood vessels in lung tissue'

MGI:6712755 (genotype) 'mice show impaired histamine-induced release of endothelial von Willebrand factor (vWF), with plasma vWF levels reduced by ~50% at 30 min after i.p. histamine injection' levels normal under baseline conditions, annotated to decreased vwf level

So changes in VWF may of may not alter blood coagulation
